packages/fxa-profile-server/lib/routes/avatar/get.js

```diff
@@ -42,20 +43,8 @@ module.exports = {
   },
   handler: async function avatar(req, h) {
     var uid = req.auth.credentials.user;
-    return db
-      .getSelectedAvatar(uid)
-      .then(avatarOrDefault)
-      .then(function (result) {
-        var rep = result;
-        if (result.id) {
-          var info = {
-            event: 'avatar.get',
-            uid: uid,
-          };
-          logger.info('activityEvent', info);
-          rep = h.response(result).etag(result.id);
-        }
-        return rep;
-      });
+    const avatar = await avatarOrDefault(uid);
```

Comment:
Does it make sense for this endpoint to get access to display or email to build out the new default avatar too? 

<img width="522" alt="Screen Shot 2021-03-30 at 11 05 19 AM" src="https://user-images.githubusercontent.com/1295288/113015286-6485d800-914b-11eb-8a11-6575269e98f5.png">

Reply:
I was hoping to avoid having to change `/avatar` since fetching the email needs a round-trip to the auth-server :(

But it seems both this and ios at least use it... so I'm looking for a way to make it better

Reply:
It gets the monogram treatment now